pkg/login/printClusterInfo.go

```diff
@@ -0,0 +1,70 @@
+package login
+
+import (
+	"fmt"
+
+	"github.com/openshift/backplane-cli/pkg/ocm"
+	logger "github.com/sirupsen/logrus"
+)
+
+//displayClusterInfo retrieves and displays basic information about the target cluster.
+
+func PrintClusterInfo(clusterID string) error {
+	logger := logger.WithField("clusterID", clusterID)
+
+	// Retrieve cluster information
+	clusterInfo, err := ocm.DefaultOCMInterface.GetClusterInfoByID(clusterID)
+	if err != nil {
+		return fmt.Errorf("error retrieving cluster info: %v", err)
+	}
+
+	// Display cluster information
+	fmt.Printf("\n%-25s %s\n", "Cluster ID:", clusterInfo.ID())
+	fmt.Printf("%-25s %s\n", "Cluster Name:", clusterInfo.Name())
+	fmt.Printf("%-25s %s\n", "Cluster Status:", clusterInfo.State())
+	fmt.Printf("%-25s %s\n", "Cluster Region:", clusterInfo.Region().ID())
+	fmt.Printf("%-25s %s\n", "Cluster Provider:", clusterInfo.CloudProvider().ID())
+	fmt.Printf("%-25s %t\n", "Hypershift Enabled:", clusterInfo.Hypershift().Enabled())
+	fmt.Printf("%-25s %s\n", "Version:", clusterInfo.OpenshiftVersion())
+	GetLimitedSupportStatus(clusterID)
+	PrintAccessProtectionStatus(clusterID)
```

Comment:
To keep the format looks better, we can change it to something like 
```
accessProtectionSatatus := PrintAccessProtectionStatus(clusterID)
...
fmt.Printf("%-25s %s\n", "Access Protection:", accessProtectionSatatus)
```

And then change the below function `func PrintAccessProtectionStatus(clusterID string)` like
```
func PrintAccessProtectionStatus(clusterID string) string
```
to return the lockbox status as value